documentation/installation.md

```diff
@@ -61,8 +66,12 @@ command:
     vagrant ssh
 
 Inside the virtual machine, set HUEY_ALWAYS_EAGER to false in
-/home/vagrant/opencraft/.env. Also, create a superuser account which will
-be used to log in to Ocim:
+/home/vagrant/opencraft/.env. Run all the pending migrations:
+
+    make manage migrate
```

Comment:
@ahmedbilal, there is a `make migrate` shortcut to run the migrations. Not to be confused with `make migrations` which translates to the `makemigrations` Django management command.

```suggestion
    make migrate
```